### ISSUE TITLE
Translation itemlist ru

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#290]
```
Translation content_itemlist.xml (r209) to russian

I used unified russian adjectives for items (with adj. crude, fine, superior 
etc) and I fixed previous translation

```

Original issue reported on code.google.com by `shell.an...@gmail.com` on 30 Dec 2011 at 3:48

Attachments:
- [content_itemlist.xml](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-290/comment-0/content_itemlist.xml)

**Comment by GoogleCodeExporter** [comment]
```
Thanks
```

Original comment by `sdeva...@gmail.com` on 30 Dec 2011 at 7:44
- Changed state: **Accepted**
- Added labels: **Type-Translation**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r213.
```

Original comment by `oskar.wi...@gmail.com` on 6 Jan 2012 at 8:47
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
